cfgov/unprocessed/js/modules/FlyoutMenu.js

```diff
@@ -65,7 +71,7 @@ function FlyoutMenu( element, triggerSel, contentSel, altTriggerSel ) {
    */
   function expand() {
     if ( !_isExpanded && !_isAnimating ) {
-      this.dispatchEvent( 'toggle', { target: this } );
+      _deferFunct = standardType.noopFunct;
```

Comment:
The queued collapse should only be called if `expand` was called and then `collapse` was called while the expand animation was underway. If expand is called and the `_deferFunct` has a value  other than noop, it means we can cancel the deferred collapse because that was set between the last call to expand and a call to collapse.